packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -181,7 +178,6 @@ function CustomDrawerContent(props: DrawerContentComponentProps<DrawerContentOpt
           />
         )}
         <View style={styles.border} />
-        {showBalances && <BalancesDisplay />}
```

Comment:
can we delete the BalancesDisplay component since it's no longer used?

Reply:
Yes, good catch :)